deployment/localup/localup.sh

```diff
@@ -174,6 +174,7 @@ function generate_genesis() {
         echo -e '[[upgrade]]\nname = "Nagqu"\nheight = 20\ninfo = ""' >> ${workspace}/.local/validator${i}/config/app.toml
         echo -e '[[upgrade]]\nname = "Pampas"\nheight = 20\ninfo = ""' >> ${workspace}/.local/validator${i}/config/app.toml
         echo -e '[[upgrade]]\nname = "Manchurian"\nheight = 20\ninfo = ""' >> ${workspace}/.local/validator${i}/config/app.toml
+        echo -e '[[upgrade]]\nname = "Hulunbeier"\nheight = 20\ninfo = ""' >> ${workspace}/.local/validator${i}/config/app.toml
```

Comment:
change height from 20 to 21 to make it different from `Manchurian` upgrade

Reply:
changed